src/table/table.component.tsx

```diff
@@ -141,9 +138,15 @@ const Table = React.memo((props: TableProps): React.ReactElement => {
     onChecked(column, false);
   };
 
+  const handleColumnWordWrapToggle = (column: string): void => {
+    const col = visibleColumns.find((col) => col.id === column);
+    if (col) col.wordWrap = !col.wordWrap;
+    setColumnOrder(columnOrder);
```

Comment:
Is this like a way of refreshing the table? What's its purpose?

Reply:
Yes, this is to refresh the table, as otherwise setting `col.wordWrap = !col.wordWrap` doesn't affect anything

Reply:
Yeah I thought so, that's quite neat. Nah I got it, thanks!